src/main/java/jenkins/plugins/mattermost/StandardMattermostService.java

```diff
@@ -83,6 +87,7 @@ public boolean publish(String message, String color) {
 				attachment.put("mrkdwn_in", mrkdwn);
 				JSONArray attachments = new JSONArray();
 				attachments.put(attachment);
+                json.put("text", text);
```

Comment:
could you fix indentation?

Reply:
definitely, I think this was an accident 😓